tests/hopr-smoke-test-node_default.cfg.yaml

```diff
@@ -0,0 +1,16 @@
+---
+hopr:
+  strategy:
+    on_fail_continue: true
+    strategies:
+      - !AutoFunding
+        funding_amount: "10000000000000000000 HOPR"
+        min_stake_threshold: "1000000000000000000 HOPR"
+      - !Aggregating
+        aggregation_threshold: 100
+        unrealized_balance_ratio: 0.9
+        aggregation_timeout: 60
```

Comment:
This is no longer used. Should be also removed from the default cfg example in hoprd I guess...